cmd/eval_wasmtarget_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// +build opa_wasm
```

Comment:
This just means that we'll miss tests if we run `go test ./cmd/` without specifying the opa_wasm tag. The CI should catch this, so I guess it's fine. The alternative would be to duplicate the tests and call into some shared code.

Reply:
Yeah it's a trade-off, I suppose. Happy to revisit this, it's not the only occurrence. 🤔